cmd/erasure-multipart.go

```diff
@@ -871,8 +876,18 @@ func (er erasureObjects) CompleteMultipartUpload(ctx context.Context, bucket str
 		}
 	}
 
+	// Hold namespace to complete the transaction
+	lk := er.NewNSLock(bucket, object)
```

Comment:
Well maybe the lock name should include upload id as well

Reply:
... but that wouldn't lock the object, then?

Reply:
> ... but that wouldn't lock the object, then?

yes, you are right, we are in complete multipart upload function. Missed that.